release.yaml

```diff
@@ -152,19 +152,19 @@ managers:
 
   - artifact: dev.galasa.liberty.manager
     version: 0.21.0
-    obr:          true
-    mvp:          true
-    bom:          true
-    javadoc:      true
-    isolated:     true
-    codecoverage: true
+    obr:          false
```

Comment:
This doesn't look right ?

Reply:
The liberty manager is completely unimplemented it looks like it was started and forgotten about - it just contains several unimplemented interfaces. It shouldn't really have been delivered or released in its current state. Thought it best to remove from everywhere. Luckily it's not mentioned on the website.

Reply:
It was however a dependency of the zos liberty manager but none of its classes were used, so I have removed it as a dependency of that manager so there are no build issues due to the liberty manager being removed from OBR.